tests/test_mail.py

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from unittest import mock
+from unittest.mock import Mock
+
+from phabricatoremails import logging
+from phabricatoremails.mail import SmtpMail, OutgoingEmail, SesMail
+
+
+MOCK_EMAIL = OutgoingEmail(
```

Comment:
This should be a fixture.

Reply:
See [this conversation](https://github.com/mozilla-conduit/phabricator-emails/pull/1#discussion_r414725433)